### ISSUE TITLE
Option type - textarea does't render , it renders as input type text

### ISSUE THREAD

**Issue by webangon** [themeum/kirki#1599]
In latest version , textarea does not render , It seems option type code has replaced it. 
```
KirKi::add_field( 'my_config', array(
	'type'        => 'code',
	'settings'    => 'meta_title',
	'label'       => esc_attr__( 'Code', 'chup' ),
	'section'     => 'general_section',

) );

KirKi::add_field( 'my_config', array(
	'type'        => 'textarea',
	'settings'    => 'txtarea',
	'label'       => esc_attr__( 'Textarea', 'chup' ),
	'section'     => 'general_section',

) );
```

![capture](https://user-images.githubusercontent.com/7946657/32285534-c7e05442-bf54-11e7-8e3d-545fe4d74818.PNG)

Also generic option type - text , the input field should strectch 100% but currently it does not. 
![capture](https://user-images.githubusercontent.com/7946657/32285821-b234dc2a-bf55-11e7-8516-c73885d83f89.PNG)

**Comment by webangon** [comment]
I don't know it's a joke or not but I deleted _class-kirki-field-text.php_  and  _class-kirki-field-textarea.php_ and now it renders perfectly and also saves the values. 

![image](https://user-images.githubusercontent.com/7946657/32286132-ae8cade0-bf56-11e7-984a-98fbb3858fc8.png)
_

**Comment by aristath** [comment]
Is that using the version from wordpress.org, or the develop branch from this repository?

**Comment by webangon** [comment]
develop branch of this repo. The same thing happens for me for version 3.0.10 of wordpress.org

**Comment by aristath** [comment]
OK, I'm still testing things here in the develop branch to make things work with WP 4.9 which will be released in a few days, so there are a few glitches at the moment. I'm planning to release an update tomorrow or the day after fixing all current issues and making the plugin compatible with WP 4.9 👍

**Comment by aristath** [comment]
Managed to figure out what was wrong here and fixed it in https://github.com/aristath/kirki/commit/9add2a935252ac4094f6a8fd16b2b2072835ba94.
This was working fine in older WP versions, obviously something changed recently and I had to remove some checks to make things work again.

As for the code control, in WP 4.9 and below code fields will be displayed as plain textareas. In WP 4.9 (scheduled to be released in less than 2 weeks) the CodeMirror library is included in WP-Core so it will work perfectly there.
This change saves more than 2MB and kirki will have a significantly smaller footprint, it can now be included in themes easier.
With the WP 4.9 release imminent I think a few days of having code fields displayed as textareas is not a big deal considering the fact that they still function fine as textfields, they just don't have syntax highlighting.
In WP 4.9 there's highlighting, error checking and a lot of other goodies implemented from the WordPress team and we just had to take advantage of all that.
